src/methods/LocalCompare.js

```diff
@@ -43,18 +43,18 @@ export default class LocalCompare extends BaseCompare {
         const png = compareData.getDiffImage().pack();
         await this.writeDiff(png, diffPath);
 
-        return this.createResultReport(misMatchPercentage, false, isSameDimensions);
+        return this.createResultReport(misMatchPercentage, false, isSameDimensions, false);
       } else {
         log(`Image is within tolerance or the same`);
         await fs.remove(diffPath);
 
-        return this.createResultReport(misMatchPercentage, true, isSameDimensions);
+        return this.createResultReport(misMatchPercentage, true, isSameDimensions, false);
       }
 
     } else {
       log('first run - create reference file');
       await fs.outputFile(referencePath, base64Screenshot, 'base64');
-      return this.createResultReport(0, true, true);
+      return this.createResultReport(0, true, true, true);
```

Comment:
although it makes sense, it feels wrong to return the results `isWithinMisMatchTolerance = true`, `isSameDimensions = true` and `isExactSameImage = true`.

Reply:
Can't argue with that.  I think, though, changing the semantics of this return could break existing tests.  If there was going to be a major (breaking) release, this could be changed.  On the other hand, since this doesn't actually do any comparisons, I'm not sure the values could be relied on for anything.  Perhaps they should be `undefined`?

Reply:
Yeah, I think we would want to keep it as is otherwise I'd probably break people's current compare checks/results when a new ref screenshot is generated.